pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -136,14 +136,15 @@ def connection_select(context, name, **options):
     """
     Select a WBEM connection definition as current or default.
 
-    Selects a connection from the persistently stored named connections if NAME
-    exists in the store to be the current connection. If the NAME argument does
-    not exist, a list of connections from the connections definition file is
-    presented with a prompt for the user to select a connection.
+    Selects the connection named NAME from the persistently stored named
+    connectionsto be the current connection if NAME exists in the store . If
+    the NAME argument does not exist, a list of connections from the
```

Comment:
Change "the NAME argument does not exist" to "the NAME argument is not specified".

Reply:
DONE